charts/pgadmin4/values.yaml

```
@@ -1,4 +1,8 @@
-# Default values for pgAdmin4.
+global:
+  # Overrides the Docker registry globally for all images.
+  imageRegistry: ""
+  # Additional global imagePullSecrets
```

Hey what do you think, would this perhaps be more informative?

```suggestion
  # Add additional image pull secrets globally.
  # Support both full format (- name: secret) and short format (- secret).
  # These will be merged with any chart-specific pull secrets.
```